modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,118 @@
+
+* With this update, the default installation namespace for {gitops-title} Operator has been changed to its own namespace called `openshift-gitops-operators`. You can still choose the old default installation namespace, `openshift-operators`, through a drop down menu available in the OperatorHub UI at installation time. You can also enable cluster monitoring on this namespace by selecting the check box, which will make the operator's performance metrics accessible within the {product-title} web console. link:https://issues.redhat.com/browse/GITOPS-3073[GITOPS-3073]
++
+[NOTE]
+====
+The {gitops-title} Operator's metrics will only be available when the operator is installed in the namespace other than `openshift-operators`.
+====
+
+* Before this update, there was no option for scaling the application controller replicas dynamically. Now, you can dynamically scale the number of application controllers based on the number of clusters managed by each application controller. link:https://issues.redhat.com/browse/GITOPS-3287[GITOPS-3287]
+
+* Before this update, there was no option for choosing an algorthim for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to round-robin, which would distribute clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
+
+* With this update, the {gitops-title} Operator will exports a few custom metrics that will allow you to track the performance of the operator. These metrics include:
+** `active_argocd_instances_total` - The number of Argo CD instances currently being managed across the cluster.
+** `active_argocd_instances_by_phase{phase="<_PHASE>"}` - The number of managed Argo CD instances in the given phase.
+** `active_argocd_instance_reconciliation_count{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The number of times the instance in a given namespace has been reconciled.
+** `controller_runtime_reconcile_time_seconds_per_instance{namespace="<_YOUR-DEFINED-NAMESPACE>"}` - The distribution of reconciliation cycles by their durations for the instance in a given namespace. link:https://issues.redhat.com/browse/GITOPS-3071[GITOPS-3071]
+
+* With this update, the Argo CD CRD API version has been upgraded from v1alpha1 to v1beta1 in order to accomodate the breaking changes resulting from the deprecation of `.spec.dex` and certain `.spec.sso` fields. To streamline the automatic migration of existing v1alpha1 ArgoCD CRs to v1beta1, conversion webhook support has been implemented. link:https://issues.redhat.com/browse/GITOPS-3040[GITOPS-3040]
```

Comment:
@masauso-lungu Please move this issue to be at the top. This issue must appear as the 1st issue in the list under the New features/Enhancements section.